curity-sgnl-kong/kong-image/configuration/kong.yml

```diff
@@ -0,0 +1,29 @@
+_format_version: '2.1'
+_transform: true
+
+services:
+- name: api
+  url: http://sgnl-kong-tutorial-api:8080/api/records
+  routes:
+  - name: api-route
+    paths:
+    - /records
+  plugins:
+  - name: phantom-token
+    config:
+      introspection_endpoint: https://sgnl-kong-tutorial-idsvr:8443/oauth/v2/oauth-introspect
+      client_id: gateway-client
+      client_secret: Password1
```

Comment:
Having these secrets embedded makes me nervous - can you placeholder them and note that they should be filled in your setup guide?

Reply:
@marc-sgnl These are already pre-set in the Curity configuration. This can be changed, however. Would it make sense to walk the user through changing that in the Curity console and updating the config?

Reply:
It was changed to have placeholders.